src/components/nav-bar/CollapsedContent.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { LocalizedLink as Link } from 'gatsby-theme-i18n/src/components/localized-link';
+import Button from '../buttons/bases/Button';
+import NavBarLink from './NavBarLink';
+import LightningImage from '../image-components/LightningImage';
+import ConsultingIllustrationImage from '../image-components/ConsultingIllustrationImage';
+import VenturesArrowImage from '../image-components/VenturesArrowImage';
+import GradientButton from '../buttons/gradient-border-buttons/GradientButton';
+
+const CollapsedContent = ({ isCollapsed, navigate }) => {
```

Comment:
Hmm bei sowas bin ich mir immer unsicher beim Naming, CollapsedContent ist ja sehr generisch - würde evtl NavBarCollapsedContent vorziehen. Weiß nicht ob das sonst durch den Pfad allein gut genug definiert ist. @cedricbals hast du evtl ne Meinung dazu?

Reply:
yeah same.

Wenn es eine Komponente ist, die nur in einer anderen Komponente liebt finde ich spezifisches Naming gut.
Wenn Collapsed contend z.B. in einem Accordion reused werden würde, wäre es fine. Aber das wirds ja nicht

Reply:
Done.